stories/components/forms/PhoneInput.stories.tsx

```diff
@@ -36,3 +38,9 @@ Placeholder.args = {
   initialCountryCode: CountryCode.AU,
   showPlaceholderExampleNumber: true,
 }
+
+HideCountryCodeAffixes.args = {
+  format: Format.LOCAL,
+  showPrefix: false,
```

Comment:
should these `show...` flags be removed if we are moving to the Format param instead?